### PR TITLE
feat(js-basics): add survey exercise

### PR DESCRIPTION
**Pull request by aclaes**

### CODE REVIEW

File: sessions/js-basics/survey-error-fixing/js/index.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+nextButton.addEventListener('click', () => {
+	console.log('Go to next question');
+});
+
+// 2. When clicking the "previous" button, this event listener is not executed. Do you know what's wrong with this code?
+const prevButton = document.querySelector('[data-js="prev-button"]');
+prevButton.addEventListener('prev', () => {
+	console.log('Go to previous question');
+});
+
```
Comment (macfrei) [inline sessions/js-basics/survey-error-fixing/js/index.js:14]: Maybe add another comment here to distinguish the following two buttons from exercise 2?
```suggestion
// ------------------ 3 & 4 -----------------
```

**Review by macfrei — APPROVED** [review]
Nice job! I like the complexity of the exercises. 👍

File: sessions/js-basics/survey-error-fixing/index.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	<head>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+		<title>Survey Error Fixing</title>
+		<link rel="stylesheet" href="./css/styles.css" />
+		<script src="./js/index.js" defer></script>
+	</head>
+	<body>
+		<div class="app">
+			<header class="header"><h1 class="title">Survey</h1></header>
```
[the remark this answers is not in the document]
Comment (aclaes) [reply sessions/js-basics/survey-error-fixing/index.html]: yes, I updated the html

File: sessions/js-basics/survey-error-fixing/index.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+				<button type="button" class="choice-button" data-js="choice-button-2">
+					Absolutely!
+				</button>
+			</main>
+			<div class="steps">
+				<button type="button" class="steps-button" data-js="prev-button">Previous</button>
+				<button type="button" class="steps-button" data-js="next-button" id="next">
+					Next
+				</button>
+			</div>
```
[the remark this answers is not in the document]
Comment (aclaes) [reply sessions/js-basics/survey-error-fixing/index.html:25]: yes, I changed the structure a bit

File: sessions/js-basics/survey-error-fixing/js/index.js (hunk 1 above)
Comment (aclaes) [reply sessions/js-basics/survey-error-fixing/js/index.js:16]: good idea. I also added a comment for 1 & 2 to be consistent

**Pull request merged**